shared.bazelrc

```diff
@@ -5,7 +5,7 @@ build --experimental_convenience_symlinks=ignore
 build --experimental_action_cache_store_output_metadata
 
 build --remote_default_exec_properties=OSFamily=darwin
-build --remote_default_exec_properties=cache_bust=macOS_12.0/3
+build --remote_default_exec_properties=cache_bust=macOS_12.0/4
```

Comment:
rules_swift incremental compilation has a cache poisoning bug where it can cache errors. This is just working around an instance of that: https://app.buildbuddy.io/invocation/106911f2-b85d-4066-a8ca-a78ca6785151?role=CI_RUNNER